docs/Networking.asciidoc

```diff
@@ -82,299 +70,52 @@ firewall is not covered in all necessary details in this documentation.
 NOTE: Another way to setup the environment with _iptables_ and _firewalld_ is described
 on the link:https://fedoraproject.org/wiki/OpenQA_advanced_network_guide[Fedora wiki].
 
-NOTE: The salt states contained by
+NOTE: Yet another way to setup multi machine tap setup is by using the salt states.
 https://github.com/os-autoinst/salt-states-openqa[salt-states-openqa] contain all the
```

Comment:
Perhaps it can be a lot simpler without repeating the same phrasing?

```suggestion
NOTE: Alternatively https://github.com/os-autoinst/salt-states-openqa[salt-states-openqa] contains
```